tests/automember/test_automember_client_context.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Test automember
+  hosts: ipaclients, ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: Include FreeIPA facts.
+    include_tasks: ../env_freeipa_facts.yml
+
+  # Test will only be executed if host is not a server.
+  - name: Execute with server context in the client.
+    ipaautomember:
+      ipaadmin_password: SomeADMINpassword
+      ipaapi_context: server
+      name: ThisShouldNotWork
+      state: rebuild
+    register: result
+    failed_when: not (result.failed and result.msg is regex("No module named '*ipaserver'*"))
+    when: ipa_host_is_client
+
+# Import basic module tests, and execute with ipa_context set to 'client'.
+# If ipaclients is set, it will be executed using the client, if not,
+# ipaserver will be used.
+#
+# With this setup, tests can be executed against an IPA client, against
+# an IPA server using "client" context, and ensure that tests are executed
+# in upstream CI.
+
+- name: Test automember using client context, in client host.
+  import_playbook: test_automember.yml
+  when: groups['ipaclients']
+  vars:
+    ipa_context: client
```

Comment:
If the context defaults to "client" on a client, this line should not be needed.

Reply:
fixed.